### PR TITLE
Fix and add documentation

### LINKED ISSUE #329: Fix Documentation
Instead of checking out `v2-latest`, we should just ask people to use `master`. `v2-latest` made sense in the beginning because `master` was unstable, but that's not the case anymore.

### PR DESCRIPTION
**Pull request by mhuzai**
Closes #329. I also took the liberty of fixing `monado-mainline.yaml`.

### CODE REVIEW

**Review by jianxiapyh — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -82,17 +82,15 @@ Visit our [web site][37] for more information.
...
-We request that you cite our following [paper][8] (new version coming soon)
-    when you use ILLIXR for a publication.
+We request that you cite our following [paper][8] when you use ILLIXR for a publication.
 We would also appreciate it if you send us a citation once your work has been published.
 
 ```
-@misc{HuzaifaDesai2020,
-    title={Exploring Extended Reality with ILLIXR: A new Playground for Architecture Research},
-    author={Muhammad Huzaifa and Rishi Desai and Samuel Grayson and Xutao Jiang and Ying Jing and Jae Lee and Fang Lu and Yihan Pang and Joseph Ravichandran and Finn Sinclair and Boyuan Tian and Hengzhi Yuan and Jeffrey Zhang and Sarita V. Adve},
+@inproceedings{HuzaifaDesai2021,
```
Comment (charmoniumQ) [inline README.md:89]:
```suggestion
@inproceedings{HuzaifaDesai2021,
  author={Huzaifa, Muhammad and Desai, Rishi and Grayson, Samuel and Jiang, Xutao and Jing, Ying and Lee, Jae and Lu, Fang and Pang, Yihan and Ravichandran, Joseph and Sinclair, Finn and Tian, Boyuan and Yuan, Hengzhi and Zhang, Jeffrey and Adve, Sarita V.},
  booktitle={2021 IEEE International Symposium on Workload Characterization (IISWC)}, 
  title={ILLIXR: Enabling End-to-End Extended Reality Research}, 
  year={2021},
  volume={},
  number={},
  pages={24-38},
  doi={10.1109/IISWC53511.2021.00014}
}
```

Comment (charmoniumQ) [reply README.md:89]: Note `volume`, `number`, `pages`, and, most importantly, `doi`.

**Review by charmoniumQ — APPROVED** [review]

**Review by JeffreyZh4ng — APPROVED** [review]
I know a few videos are linked that bring you to a video under the YT channel, but should we mention that you can also find the consortium recordings on the YT too?

**Comment by mhuzai** [comment]
> 

We link to the consortium, so I think that's sufficient.

**Pull request merged**